docs/src/SearchField.doc.js

```diff
@@ -2,13 +2,55 @@
 
 import type { Node } from 'react';
 import PropTable from './components/PropTable.js';
-import Example from './components/Example.js';
 import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
 
 const cards: Array<Node> = [];
 const card = (c) => cards.push(c);
 
-card(<PageHeader name="SearchField" />);
+card(
+  <PageHeader
+    name="SearchField"
+    defaultCode={`
+function SearchFieldExample() {
+  const [value, setValue] = React.useState('');
+
+  return (
+    <Box padding={3} display="flex" alignItems="center" flex="grow">
+      <Box padding={3}>
```

Comment:
nit: Flex could make this markup cleaner, no need for most of these wrapping Boxes